packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -219,6 +222,7 @@ function useBlockEditorSettings( settings, postType, postId ) {
 			__experimentalBlockPatterns: blockPatterns,
 			__experimentalBlockPatternCategories: blockPatternCategories,
 			__experimentalUserPatternCategories: userPatternCategories,
+			__experimentalUserPatterns: userPatterns,
```

Comment:
Maybe I'm missing something but the selector was already in block-editor. Meaning it was already a "shared" selector somehow. So I'm wondering why we're adding this new config that probably duplicates some settings that are already there.

Reply:
`getUserPatterns` in the block editor selectors is currently just a private method that is used by the `getAllAllowedPatterns` selector, so not shareable with site editor currently. 

To minimise any backwards compat issues with changing existing selectors in `block-editor`, and moving away from the use of ReusableBlocks terminology it seemed like a good option was to create new `Patterns` based selectors in `core-data` and deprecate any selectors that are no longer used  in `block-editor`. 

However, given that `block-editor is already a dependency of `edit-site` another option is to just modify the selectors in the block editor store and use those in the site editor rather than adding completely new ones in `core-data`. I can take a look at doing that if you think that is a better approach.

Reply:
what concerns me the most is that the block editor is already aware of these pattern in its settings and now we're adding another setting that basically is just another format of an existing one.

Reply:
Makes sense, will reconsider in light of that.